### PR TITLE
JuneJuly 2022 Newsletter

### PR DESCRIPTION
**Pull request by Robadob**
Norbert has confirmed his changes, so should be ready for review.

### CODE REVIEW

File: _newsletters/2022-07-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+#### LunchBytes needs YOU!
+LunchBytes are organised by and for the research software community at The University of Sheffield. If you'd like to curate a session on a topic or present something, get in touch by emailing [lunchbytes-organisers-group@sheffield.ac.uk](mailto:lunchbytes-organisers-group@sheffield.ac.uk) - Or suggest topics [on the jamboard](https://jamboard.google.com/d/1-51cRf0pwZl8O10CnLeJGAqKcnbww-QGaYjszFK-H38/).
+
+###  Research IT Forum
+Research IT Forum is primarily to engage the research computing community and share research carried out in specific departments. We also use the Forum to signpost new HPC resources. Previous Fora included GPU accelerated and AI/Machine Learning research, image processing and analysing human language. The recordings are available for all attendees. The next Forum is targeted for October. 
+
+### Events
+* [DiveRSE talk series: Lessons from Diversity, Equity and Inclusion in open source software](https://diverse-rse.github.io/events/2022-07-05) 16:00-17:00 5th July
```
Comment (Robadob) [inline _newsletters/2022-07-05-newsletter.md:29]: Note, this event should be pulled if newsletter goes out later than today.

File: _newsletters/2022-07-05-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+### Support
+#### Code Clinics
+Why not come to a [Code Clinic](https://docs.google.com/forms/d/e/1FAIpQLScGXS55qjU0D0Zcz-KHOVcNTahcr3YC3H0OpoKBo3lWXWED5A/viewform)? We're keen to hear from you.
+
+Code Clinics are fortnightly supported sessions run by the RSE team and IT Services’ [Research and Innovation IT](https://www.sheffield.ac.uk/it-services/research) (ITS R&I) team. They are open to anyone at TUoS writing code for research to get help with programming problems and general advice on best practices.
+
+At each session, members of the RSE and/or ITS R&I teams will be available to review code, advise, troubleshoot, and suggest ways to improve your computational workflows.
+
+#### Research IT HPC Drop In
+HPC Drop-In sessions are providing assistance with HPC related user issues such as challenges in scaling an application from desktop to supercomputer. We are considering extending the number of our sessions to two or three weekly. These interactive sessions could provide a better interface with our users than our non-interactive ticketing system.
```
Comment (ns-rse) [inline _newsletters/2022-07-05-newsletter.md]: Is there a link detailing when the drop-in sessions are? I had a quick search through the link below (https://students.sheffield.ac.uk/it-services/research) but nothing jumped out.

**Review by ns-rse — APPROVED** [review]
Very minor suggestion for a link to help people find HPC Drop-in sessions if they are documented somewhere, otherwise good to go.

File: _newsletters/2022-07-05-newsletter.md (hunk 2 above)
Comment (Robadob) [reply _newsletters/2022-07-05-newsletter.md]: Will drop norbert an email.

Comment (Robadob) [reply _newsletters/2022-07-05-newsletter.md]: > We invite people when the topdesk call is not appropriate. 
> Details are also sent out by the HPC mailing list.

File: _newsletters/2022-07-05-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+* [A Researcher Guide to Writing a Climate Justice Oriented Data Management Plan](https://zenodo.org/record/6451499)
+
+### Opportunities
+* [Enabling human centred decision making through data visualisation Grant](https://www.ukri.org/opportunity/enabling-human-centred-decision-making-through-data-visualisation/) - Intention to Submit Deadline 8th July
+* [UK Gov Knowledge transfer partnerships (KTP): 2022 to 2023 Round 3 Funding Competition](https://apply-for-innovation-funding.service.gov.uk/competition/1222/overview/63827061-eb42-4f00-855a-319f88d2a41f) - Closes 14th September
+
+### Research IT news
+#### A100 GPU nodes
+Bessemer is now [temporarily upgraded with 64 A100 GPUs](https://changelog.hpc.shef.ac.uk/bessemer-a100-nodes/). The resource is available for immediate use, however, it will be moved to Stanage in due course. For more information please visit [https://docs.hpc.shef.ac.uk/en/latest/bessemer/GPUComputingBessemer.html#gpucomputing-bessemer](https://docs.hpc.shef.ac.uk/en/latest/bessemer/GPUComputingBessemer.html#gpucomputing-bessemer).
```
Comment (ptheywood) [inline _newsletters/2022-07-05-newsletter.md]: Stanage is not widely known yet / is introduced below, so at this point in the newsletter it is just a place in the peaks, which is an odd place for A100's to live.

Probably worth either reversing the subseciton order, or clarifying that they will vbe moved to the/our `new HPC facility (Stanage)`, maybe even with a `see below` mixed in.

**Review by ptheywood — CHANGES_REQUESTED** [review]

**Review by ptheywood — APPROVED** [review]

**Pull request merged**